scripts/docs/build-api-docs.js

```diff
@@ -11,4 +11,7 @@
 
 const buildApiDocs = require( './buildapi' );
 
-buildApiDocs();
+buildApiDocs()
+	.catch( () => {
+		process.exitCode = -1;
```

Comment:
Not sure about negative error codes. But I guess that everything despite exitCode that is equal to 0 will be treated as an error.

Reply:
<img width="483" alt="screen shot 2017-12-11 at 11 49 06" src="https://user-images.githubusercontent.com/2768047/33827946-62aea238-de6a-11e7-8d44-a4d922c7f9e0.png">

Reply:
You're right. We should use `1` error code (http://tldp.org/LDP/abs/html/exitcodes.html).

`1` means _Catchall for general errors_.